docs/MapColonies/DEM/Services/elevation/README.md

```diff
@@ -0,0 +1,89 @@
+---
+id: dem-elevation-service
+slug: info
+title: Elevation ⛰️
+description: Service to return geographic coordinate [lat, lon] elevation 
+tags:
+  - dem
+  - API
+  - elevation
+  - height
+  - coordinate
+---
+
+# Elevation Service ⛰️
+
+:::info
+Click [here](/docs/MapColonies/dem/services/elevation/api) for the OpenAPI
+:::
+
+## Purpose
+The service lets users to recieve elevation(height) info at a given location(s)/position(s) [lon,lat] with/without extra data eg. resolution, DTM/DSM, etc.
```

Comment:
Please change to:
Allows users to request elevation (height) information at given location(s)/position(s) in [lon,lat] format with extra data eg. resolution, DTM/DSM, etc.

Reply:
DONE